gnd/src/main/java/com/google/android/gnd/repository/ProjectRepository.java

```diff
@@ -137,4 +139,8 @@ public void clearActiveProject() {
     localValueStore.clearLastActiveProjectId();
     activateProjectRequests.onNext(Optional.empty());
   }
+
+  public Completable saveUser(User user) {
```

Comment:
This class is scoped specifically to Project - could you please create a UserRepository and add the method there instead?

Reply:
Done